syntax = "proto3";
package api;

service AccountsApi {
    rpc GetAccountInfo (AccountReq) returns (AccountRsp) {
    }
}

message AccountReq {
    string accountCode = 1;
}
message AccountRsp {
    string accountInfo = 1;
}